### DISCUSSION TITLE
Modifications to the fluent messages

### DISCUSSION THREAD

**Discussion by pilgi3** [PSAppDeployToolkit/PSAppDeployToolkit#1137] [Ideas]
It would be great, if it is possible to modify the install progress message.
For example the color of the left bar and the icon in the bar.
![image](https://github.com/user-attachments/assets/08d95aed-85db-4292-821e-239baf510f32)

**Comment by mjr4077au** [comment]
Transferred to https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/issues/1164.
